lib/util/viewer.js

```diff
@@ -136,11 +136,8 @@ export function getStopTimesByPattern(stopData) {
           times: []
         }
       }
-      // Exclude the last stop, as the stop viewer doesn't show arrival times to a terminus stop.
-      const filteredTimes = times.filter(excludeLastStop)
```

Comment:
This change results in the display of arrival times (in addition to departures) at terminal stops. Per our discussion,. this is not desired.

Reply:
I can't replicate this. Do you have any examples?

Reply:
Ok never mind I can see it on some bus routes where the last stop of one pattern is not identical to the first stop of another. Looking at the behavior I am not against that